code/obj/flock/structure/sentinel.dm

```diff
@@ -0,0 +1,93 @@
+//
+//Sentinel structure,
+//
+/obj/flock_structure/sentinel
+	name = "Glowing pylon"
+	desc = "A glowing pylon of sorts, faint sparks are jumping inside of it."
+	icon_state = "sentinel"
+	flock_id = "Sentinel"
+	health = 80
+	var/charge_status = -1 //-1 == not charged,0 == losing charge, 1 == charging, 2 == charged
+	var/charge = 0 //0-100 charge percent
+	var/powered = 0
+
+	event_handler_flags = USE_CANPASS
+	passthrough = 1 //drones can pass through this, might change this later, as balance point
+
+	usesgroups = 1
+	poweruse = 20//debug amount scale up if needed.
+
+/obj/flock_structure/sentinel/New(var/atom/location, var/datum/flock/F=null)
+	..(location, F)
+	src.filters = filter(type="rays", x=-0.2, y=6, size=1, color=rgb(255,255,255), offset=rand(1000), density=20, threshold=0.2, factor=1, flags=FILTER_UNDERLAY)
```

Comment:
```suggestion
	src.filters = filter(type="rays", x=-0.2, y=6, size=1, color=rgb(255,255,255), offset=rand(1000), density=20, threshold=0.2, factor=1)
```

according to the docs, this should work the same as long as the sentinel isn't a "plane master" (whatever that is)

Reply:
ill just leave the flag on, the linter issue is supposedly fixed, so ill just hope the version we use is updated. If by the time this pr is due to be merged and the issue still lingers ill remove the flag